psi4/src/psi4/libmints/deriv.cc

```diff
@@ -714,7 +680,8 @@ SharedMatrix Deriv::compute()
     }
 
     // Obtain nuclear repulsion contribution from the wavefunction
-    SharedMatrix enuc(new Matrix(molecule_->nuclear_repulsion_energy_deriv1()));
+    std::vector<double> field_strength = wfn_->get_dipole_field_strength();
+    SharedMatrix enuc(new Matrix(molecule_->nuclear_repulsion_energy_deriv1(field_strength)));
```

Comment:
`SharedMatrix enuc = std::make_shared<Matrix>(molecule_->nuclear_repulsion_energy_deriv1(field_strength));` just because I was on a recent `make_shared` rampage :wink:

Reply:
Good catch!  Will change this...